### PR TITLE
Removal of Istio component (#2836)

### LINKED ISSUE #2836: [FEATURE REQUEST] Removal of Istio component
**Is your feature request related to a problem? Please describe.**
We are considering replacement of Istio component with different solutions and Istio causes problems during upgrade of Kubernetes.

**Describe the solution you'd like**
Remove Istio role from Epiphany.

**Describe alternatives you've considered**
Replace Istio with different service mesh solution.

**Additional context**
None.

---

**DoD checklist**

* [x] Changelog updated
* [x] COMPONENTS.md updated / doesn't need to be updated
* [x] Schema updated / doesn't need to be updated
* [ ] Feature has automated tests
* [x] Automated tests passed (QA pipelines)
  * [x] apply
  * [ ] upgrade
* [x] Idempotency tested
* [x] Documentation added / updated / doesn't need to be updated
* [x] All conversations in PR resolved
* [x] Solution meets requirements and is done according to design doc
* [x] Usage compliant with license
* [x] Backport tasks created / doesn't need to be backported

### PR DESCRIPTION
**Pull request by cicharka**
Tested `epicli apply` on Azure cluster with Ubuntu machines

### CODE REVIEW

File: .devcontainer/Dockerfile [hunk 1]
```diff
@@ -33,11 +32,6 @@ RUN : INSTALL HELM BINARY \
  && chmod +x ./kubectl \
  && mv ./kubectl /usr/local/bin/kubectl \
  && kubectl version --client \
```
Comment (atsikham) [inline .devcontainer/Dockerfile]:
```suggestion
 && kubectl version --client
```

Comment (atsikham) [reply .devcontainer/Dockerfile]: Check the same in another Dockerfile

File: docs/home/howto/ISTIO.md [hunk 2]
```diff
@@ -1,115 +0,0 @@
-## Istio
```
Comment (atsikham) [inline docs/home/howto/ISTIO.md:1]: Is there a reference to this file, HOWTO.md for example?

**Review by atsikham — CHANGES_REQUESTED** [review]

File: docs/home/howto/ISTIO.md (hunk 2 above)
Comment (cicharka) [reply docs/home/howto/ISTIO.md:1]: I removed all of the references to Istio in our docs (apart from design docs which will be cleared in another task).

**Review by erzetpe — DISMISSED** [review]

**Review by atsikham — DISMISSED** [review]

**Review by erzetpe — APPROVED** [review]

**Review by asciible — APPROVED** [review]

**Pull request merged**